backend/tests/integration-async/oai/test_views.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from time import sleep
+
+from helpers.providers.faker import faker
+from inspire_dojson.api import record2marcxml
+from invenio_db import db
+from invenio_oaiserver.models import OAISet
+
+from inspirehep.records.api import LiteratureRecord
+
+
+def test_oai_with_for_cds_set(
+    inspire_app, celery_app_with_context, celery_session_worker
+):
+    data = {"_export_to": {"CDS": True}}
+    record_data = faker.record("lit", data)
+    record = LiteratureRecord.create(record_data)
+    record_marcxml = record2marcxml(record)
+    db.session.commit()
+
+    set_name = inspire_app.config["OAI_SET_CDS"]
+    oaiset = OAISet(spec=f"{set_name}", name="Test", description="Test")
+    oaiset.search_pattern = f"_oai.sets:{set_name}"
```

Comment:
why do you need this? I thought the `search_pattern` was used to put records in collections (percolator stuff), not to find records already assigned to collections.

Reply:
Indeed we don't need it